packages/react-ui/components/Select/__tests__/Select-test.tsx

```diff
@@ -189,6 +189,29 @@ describe('Select', () => {
     expect(screen.queryByRole('button', { name: ninth })).not.toBeInTheDocument();
   });
 
+  it('should clear the value when null passed', () => {
+    const Comp = () => {
+      const items = ['One'];
+
+      const [value, setValue] = useState<string | null>('One');
+
+      return (
+        <>
+          <Select<string | null> items={items} value={value} onValueChange={setValue} />
+          <button onClick={() => setValue(null)}>Clear</button>
+        </>
+      );
+    };
+
+    render(<Comp />);
+
+    const input = screen.getByText('One');
+    expect(input).toHaveTextContent('One');
+
+    userEvent.click(screen.getByRole('button', { name: 'Clear' }));
+    expect(input).toHaveTextContent('Ничего не выбрано');
```

Comment:
Текст `Ничего не выбрано` тоже константа:
```suggestion
    expect(input).toHaveTextContent(SelectLocaleHelper.get(defaultLangCode).placeholder as string);
```

Reply:
[Поправил](https://github.com/skbkontur/retail-ui/pull/2987/commits/8959ce68e7c0d40983a6eb76b213b0db1b36894f)